kubeshop/monokle
# fix: Kubeconfig path not changing

This PR...

## Changes

-  Under KUBECONFIG, *Browse Button* and *spinner* gets toggles based on the file that gets uploaded.

## Fixes
#2070 

## How to test it

- Create a project
- Select three dot in top right corner.
- Select setting and then under *Active Project* Browse file in KUBECONFIG
- You can see there will be spinner till the file gets uploaded

## Screenshots
- #### Before uploading the file
![image](https://user-images.githubusercontent.com/35147480/178782855-0cb437a7-8ac5-405b-8df1-44a10ac26c3c.png)

- ####  After selecting a file
![image](https://user-images.githubusercontent.com/35147480/178782984-3d162726-3127-4f07-90a9-338072c5af12.png)

- ####  After file gets uploaded
![image](https://user-images.githubusercontent.com/35147480/178783472-8c2f1292-2b70-4fee-b13b-2431fdf0bb89.png)

## Checklist
